server/lib/AirspaceChecker.js

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable no-console */
+
+const querystring = require('querystring');
+const fetch = require('isomorphic-fetch');
+
+class AirspaceChecker {
+  /**
+   * Determines if the drone is allowed to fly on this flight path
+   * 
+   * @param {*} origin Array of 2 values: latitude & longitude
+   * @param {*} destination Array of 2 values: latitude & longitude
+   * @returns {boolean} True if the drone can fly on this segment, false if not
+   */
+  static async checkSpace(origin, dest) {
+    let isOK;
+
+    try {
+      const params = {
+        geometry: `{ "type": "LineString", "coordinates": [[${origin[1]}, ${origin[0]}], [${dest[1]}, ${dest[0]}]] }`,
+        types: 'airport',
+        geometry_format: 'geojson',
+        full: true,
+        buffer: 3,
```

Comment:
What's this buffer thing?

Reply:
Buffer is a part of the Airmap API that specifies a "buffer" range in meters for the drone path to account for wind and navigation variance.

Reply:
👍

Reply:
Why is it set to 3?

Reply:
Seemed good and there's nothing we have in the DB or otherwise to suggest that it should be more or less.  If at a later point we have a chance to work it into drone type or something we'll change it or make it a variable but until then I'm leaving it as-is.